tensorflow/core/ops/image_ops.cc

```diff
@@ -358,9 +358,9 @@ try_recover_truncated:  If true try to recover an image from truncated input.
 acceptable_fraction: The minimum required fraction of lines before a truncated
   input is accepted.
 dct_method: string specifying a hint about the algorithm used for
-  decompression.  Defaults to "" which maps to a system-specific
-  default.  Currently valid values are ["INTEGER_FAST",
-  "INTEGER_ACCURATE"].  The hint may be ignored (e.g., the internal
+  decompression.  Defaults to '' which maps to a system-specific
+  default.  Currently valid values are ['INTEGER_FAST',
+  'INTEGER_ACCURATE'].  The hint may be ignored (e.g., the internal
```

Comment:
For these three lines, it looks to me like the string constants should be additionally enclosed in backticks (to make it look like this: `''` etc.).